Fantasy-Quest/Assets/Scripts/UI/SlideshowParent.cs

```diff
@@ -116,6 +137,7 @@ private IEnumerator ShowSlide(SlideStruct slide)
         [Button]
         public void StartSlideshow()
         {
+            playerInput.Player.Skip.performed += SkipCutscene;
```

Comment:
Do It in `OnEnable`

Reply:
Could i use `playerInput.Enable()` outside of  `OnEnable`,  ?